Source/ALDataRequestView.swift

```diff
@@ -141,6 +139,7 @@ public class ALDataRequestView: UIView {
         loadingView?.removeFromSuperview()
         emptyView?.removeFromSuperview()
         reloadView?.removeFromSuperview()
+        hidden = true
```

Comment:
Waarom hier hidden = true en die andere weghalen? Dit is een breaking change en zorgt er dus voor dat de DataRequestView niet meer op hidden wordt gezet zodra er niets geladen wordt of iets.